backend/services/rclone_service.py

```
@@ -0,0 +1,102 @@
+            try:
+                with os.fdopen(fd, 'w') as f:
+                    f.write(config_content)
+                
+                self._config_file = config_path
+                
+                # Set environment variable for rclone to find the config
+                os.environ["RCLONE_CONFIG"] = config_path
+                
+                logger.info(f"Rclone config loaded and written to {config_path}")
+                self._config_loaded = True
+                return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)